src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form__big {
+  margin-top: 20px;
+  position: relative;
+}
+
+.form__small {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search {
+  font-family: inherit;
+  width: 100%;
+  height: 70px;
+  line-height: 22px;
+  padding-left: 62px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search--small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  font-weight: bold;
+  outline: none;
+}
+
+.search--small::placeholder {
+  font-size: inherit;
+}
```

Comment:
why you need this if you have `font-size` in `.search--small` class. It's redundant